src/main/scala/cromwell/engine/ExecutionStatus.scala

```diff
@@ -2,11 +2,11 @@ package cromwell.engine
 
 object ExecutionStatus extends Enumeration {
   type ExecutionStatus = Value
-  val NotStarted, Starting, Running, Failed, Done, Aborted = Value
+  val NotStarted, Starting, Running, Failed, Preempted, Done, Aborted = Value
 
   implicit class EnhancedExecutionStatus(val status: ExecutionStatus) extends AnyVal {
     def isTerminal: Boolean = {
-      Seq(Failed, Done, Aborted) contains status
+      Seq(Failed, Done, Aborted, Preempted) contains status
```

Comment:
Which is a prime example of why it's bad to use `Seq()`. This whole time it *was* a `List` but you're saying it shouldn't be.

(I don't really care what it is, just that we're not creating Seqs willy nilly)